modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/configuration/persistence/listener/MFASystemConfigurationModelListener.java

```diff
@@ -95,9 +94,7 @@ private void _sendNotificationToInstanceAdministrators(
 		Role role = _roleLocalService.getRole(
 			companyId, RoleConstants.ADMINISTRATOR);
 
-		for (User user : _userLocalService.getRoleUsers(role.getRoleId())) {
-			long userId = user.getUserId();
-
+		for (long userId : _userLocalService.getRoleUserIds(role.getRoleId())) {
 			_userNotificationEventLocalService.sendUserNotificationEvents(
 				userId, ConfigurationAdminPortletKeys.INSTANCE_SETTINGS,
 				UserNotificationDeliveryConstants.TYPE_WEBSITE, false,
```

Comment:
@marianoalvarosaiz, the commit message looks wrong, I think it should avoid getting User model.